MAINTAINERS.md

```diff
@@ -1,9 +1,11 @@
 # Core Maintainers of this repository
 
-| Name                | Email                 | GitHub                                   |
-|---------------------|-----------------------|------------------------------------------|
-| Bartłomiej Płotka   | bartek@improbable.io  | [@bwplotka](https://github.com/bwplotka) |
-| Dominic Green       | dom@improbable.io     | [@domgreen](https://github.com/domgreen) |
+| Name                | Email                 | Slack       | GitHub                                                     |
+|---------------------|-----------------------|-------------|------------------------------------------------------------|
+| Bartłomiej Płotka   | bartek@improbable.io  | `@bwplotka` | [@bwplotka](https://github.com/bwplotka)                   |
+| Dominic Green       | dom@improbable.io     | `@domgreen` | [@domgreen](https://github.com/domgreen)                   |
+| Frederic Branczyk   | ?                     | `@brancz`   | [@brancz](https://github.com/brancz)                       | 
```

Comment:
@brancz I need your signature... here (: What's the email you want to use?

Reply:
fbranczyk@gmail.com